sources/AppBundle/Event/Form/TicketType.php

```diff
@@ -173,6 +173,21 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ;
         });
 
+        if ($event->getTransportInformationEnabled()) {
+            $builder->add('transportMode', ChoiceType::class, [
+                'label' => 'Quel est votre mode de transport ?',
+                'required' => true,
+                'choices' => array_flip(Ticket::TRANSPORT_MODES),
```

Comment:
en local j'ai une 500 à cause de ce warning "Warning: array_flip(): Can only flip STRING and INTEGER values!". En enlevant le null => null, il n'y a plus l'erreur. ne faudrait-il pas l'enlever de la constante et faire un merge ou + sur le tableau pour ajouter le null ?

Reply:
Effectivement, un changement de dernière minute. J'en ai profité pour ajouter un test fonctionnel.